src/features/core/cookieConsent/ketch/browser.js

```diff
@@ -0,0 +1,83 @@
+import logger from '../../../../utils/logUtil';
+
+/* eslint-disable class-methods-use-this */
+class Ketch {
+  constructor() {
+    this.userConsentedPurposes = [];
+    this.userDeniedPurposes = [];
+
+    // updateKetchConsent callback function to update current consent purpose state
+    // this will be called from ketch ruddersatck plugin
+    window.updateKetchConsent = (consent) => {
+      if (consent.purposes) {
+        const p = consent.purposes;
+        this.userConsentedPurposes = [];
+        this.userDeniedPurposes = [];
+        Object.entries(p).forEach((e) => {
```

Comment:
what is the type of consent.purposes entries? can we denote what is the e[0] and what is the e[1]  for the variables to be self explanatory please?